src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -49,28 +49,34 @@ <h1 class="h1" id="title">
     </div>
 </div>
 
-<h1 class="h1" id="overview">{% trans "Persoonlijk overzicht" %}</h1>
-
 <div class="tabled tabled--flexible personal-overview">
+    <h2 class="h2" id="overview">{% trans "Overzicht" %}</h2>
     {% if request.user.is_digid_and_brp and view.config.my_data %}
         <div class="tabled__row">
-            <div class="tabled__item tabled__item--bold">{% trans "Mijn gegevens" %}</div>
+            <div class="tabled__item tabled__item--bold">{% link href="profile:categories" text=_('My details') %}</div>
             <div class="tabled__item">{{request.user.get_full_name}}{% if request.user.get_address %}, {{ request.user.get_address }}{% endif %}</div>
             <div class="tabled__item tabled__item--force-right tabled__item--mobile-big">{% link href="profile:data" text="Bekijken" icon="arrow_forward" icon_position="after" primary=True %}</div>
         </div>
     {% endif %}
 
     {% if view.config.selected_categories %}
         <div class="tabled__row">
-            <div class="tabled__item tabled__item--bold">{% trans "Interessegebieden" %}</div>
+            <div class="tabled__item tabled__item--bold">{% link href="profile:categories" text=_('My categories') %}</div>
```

Comment:
Why English? Most of the texts in this template are in Dutch.

Reply:
I was thinking: if I have to change some text per request from one of the municipalities, or add completely new text, the default should be in English - this could be a gradual way to change all code-text into English. I can update this `me.html` template to be completely English.

Reply:
Good to know. Feel free to get started with this, but no pressure.